scripts/add_electricity.py

```diff
@@ -385,6 +385,36 @@ def attach_wind_and_solar(
                 efficiency=costs.at[suptech, "efficiency"],
             )
 
+            if tech == "csp":  # add store and link for CSP
+
+                n.madd(
+                    "Store",
+                    ds.indexes["bus"],
+                    " " + tech,
+                    bus=ds.indexes["bus"],
+                    carrier=tech,
+                    e_nom_extendable=True,
+                    e_cyclic=True,
+                    capital_cost=costs.at[tech, "capital_cost"],
+                )
+
+                available_csp_buses_i = n.buses.index[
+                    n.buses.index.isin(ds.indexes["bus"])
+                ]
+
+                n.madd(
+                    "Link",
+                    ds.indexes["bus"],
+                    " " + tech,
+                    bus0=ds.indexes["bus"],
+                    bus1=available_csp_buses_i,
```

Comment:
Not sure I get it: aren't length values of `ds.indexes["bus"]` and `available_csp_buses_i` different?

Reply:
The length for `ds.indexes["bus"]` is 161, while the length of `n.buses.indexes` is 203. Using n.buses.indexes results in a broadcast error, so I first filter for only indexes available in both buses before applying it into the link.

Reply:
This has been removed as we are no longer handling this section in the `add_electricity.py` script